### PR TITLE
Create documentation on creating full dumps for troubleshooting.

### PR DESCRIPTION
**Pull request by nathanctech**
Hopefully easy to follow.

### CODE REVIEW

File: content/support/server-debug.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    This registers procdump as a debugger to capture certain crashes.
+
+5) Open task manager, click "Details". Locate FXServer.exe. There should be a "PID" column. Note down the number.
+
+6) Go back to your command prompt and type in: `procdump64.exe -accepteula -e 1 -h -ma pidhere`
+
+where `pidhere` is the number you found from step 4. If you get an error, reopen command prompt as administrator and try again.
+
+7) Wait for server to crash. When it does, it will write a large .dmp file to the procdump folder. Zip up this file and upload to google drive/dropbox and supply to #server-bugs or where it was requested.
```
Comment (cerium-cfx) [inline content/support/server-debug.md]: `#server-bugs` is ambiguous.

Perhaps say `to the #server-bugs channel on [Discord](<discord_invite_link_here>)` or similar, instead. Don't forget that bug reports can also be made on the FiveM forum. Mention that too with a link

File: content/support/_index.md [hunk 2]
```diff
@@ -6,3 +6,4 @@ weight: 800
 - [Client FAQ](/support/client-faq)
 - [Client issues](/support/client-issues)
 - [Server issues](/support/server-issues)
+- [Server Debugging](/support/server-debug)
```
Comment (cerium-cfx) [inline content/support/_index.md]: A little nitpicking, but `Debugging` should be lower case `debugging` to confirm the rest of the docs

File: content/support/server-debug.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+---
+title: Server Debugging
```
Comment (cerium-cfx) [inline content/support/server-debug.md]: Same as previous nitpicking.

**Comment by nathanctech** [comment]
>It would also be neat to have a section for "Read Dumps" or so, but perhaps that's another PR.

Agreed. I'll work on a basic write-up with some examples of common crashes and their cause. Bit more research there!

**Comment by nathanctech** [comment]
For some reason github says changes are requested, but I've fixed everything that's been pointed out. Should be okay now.

**Pull request merged**